packages/tocco-ui/src/Button/StyledButton.js

```diff
@@ -45,15 +45,15 @@ const meltButtons = props => {
 const declareButtonColor = props => {
   let declareColor
   const {ink, look} = props
-  const {FLAT, RAISED} = stylingLook
+  const {BALL, FLAT, RAISED} = stylingLook
   const {BASE, PRIMARY} = stylingInk
   if (look === FLAT && ink === BASE) {
     declareColor = generateFlatBaseColors
   } else if (look === FLAT && ink === PRIMARY) {
     declareColor = generateFlatPrimaryColors
-  } else if (look === RAISED && ink === BASE) {
+  } else if ((look === BALL || look === RAISED) && ink === BASE) {
```

Comment:
if would define the (look === FLAT && ink === BASE) case as default? Because i think declareColor should always be defined otherwise the app will crash on line 59